### PR TITLE
Use Stargate 1.0.26

### PR DESCRIPTION
**Pull request by dimas-b**

### CODE REVIEW

File: cassandra-3.11/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8u252-slim
 
-ARG STARGATE_VERSION=v1.0.25
+ARG STARGATE_VERSION=1.0.26
```
Comment (dougwettlaufer) [inline cassandra-3.11/Dockerfile:3]: These all need the "v" prefix

Comment (dimas-b) [reply cassandra-3.11/Dockerfile:3]: a-ha... I use the new update script... probably needs fixing :)

Comment (dimas-b) [reply cassandra-3.11/Dockerfile]: It was my mistake in specifying the script's arg :facepalm: ... fixed.

**Review by dougwettlaufer — APPROVED** [review]

File: cassandra-3.11/Dockerfile (hunk 1 above)
Comment (ivansenic) [reply cassandra-3.11/Dockerfile]: I wanted to make script accept a tag like value.

**Review by mpenick — APPROVED** [review]

**Pull request merged**